src/main/java/com/celements/model/access/DefaultModelAccessFacade.java

```diff
@@ -358,8 +358,8 @@ public BaseObject getXObject(XWikiDocument doc, DocumentReference classRef, Stri
     return Collections.unmodifiableMap(ret);
   }
 
-  private void checkNotTranslation(XWikiDocument doc) {
-    String msg = "Trying to access XObjects on translation '{0}' of doc '{1}";
+  void checkNotTranslation(XWikiDocument doc) {
+    String msg = "Trying to access XObjects on translation ''{0}'' of doc ''{1}''";
```

Comment:
for better readability. You may inline instead, but do not assign a constant final string to a local variable.